.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
+poetry install --with dev
+pip install pre-commit
```

Comment:
Wouldn't this be already installed with the `poetry install --with dev` command?

Reply:
In the past when I've tried to use `poetry run pre-commit install --install-hooks` instead of `pip install pre-commit; pre-commit install --install-hooks`  it has ended up breaking VSCode's git integration. I could use the CLI just fine, but had issues whenever I tried to use the VSCode user interface. 

I'll go ahead and test if that limitation still exists or if I can figure out a way around it.